src/controller/CommissioningDelegate.h

```diff
@@ -81,7 +80,10 @@ enum CommissioningStage : uint8_t
     kPrimaryOperationalNetworkFailed, ///< Indicate that the primary operational network (on root endpoint) failed, should remove
                                       ///< the primary network config later.
     kRemoveWiFiNetworkConfig,         ///< Remove Wi-Fi network config.
-    kRemoveThreadNetworkConfig        ///< Remove Thread network config.
+    kRemoveThreadNetworkConfig,       ///< Remove Thread network config.
+    kGetTCAcknowledgments,            ///< Waiting for the higher layer to provide terms and conditions acknowledgements.
+    kConfigureTCAcknowledgments,      ///< Send SetTCAcknowledgements (0x30:6) command to the device
+    kCleanup,                         ///< Call delegates with status, free memory, clear timers and state
```

Comment:
The comments above claim this value cannot be moved later in the list.  Either it _can_ and the comments need fixing, or it cannot and should not be.

Reply:
The comment was originally added 2 years ago. When I had left the enum as it was, I was seeing a cirque test failure because some of the commissioning tests were attempting to validate that the stage did not exceed the cleanup. After adding kConfigureTCAcknowledgments after kCleanup I was seeing this error. After I moved it before kCleanup I found that the check worked appropriately.

After making the change, we can see that the cirque tests are able to succeed after re-ordering the enum. Basically, there was some checks that depended on mutual knowledge of the (internal) enum order, between some python code and the implementati0on, but that is no longer the case.